content/docs/2.3/scalers/graphite.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "Graphite"
+layout = "scaler"
+availability = "v1.0+"
+maintainer = "Community"
+description = "Scale applications based on Graphite."
+go_file = "graphite_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `graphite` trigger that scales based on a Graphite.
+
+```yaml
+triggers:
+- type: graphite
+  metadata:
+    # Required
+    serverAddress: http://<graphite-host>:81
+    metricName: request-count
+    query: stats.counters.http.hello-world.request.count.count # Note: query must return a vector/scalar single element response
+    threshold: '100'
+    queryTime: '-10Minutes'
+```
+
+**Parameter list:**
+
+- `serverAddress` - Address of Graphite
+- `metricName` - Metric name to use
+- `query` - Query to run
```

Comment:
Can you expand this a bit please? I see the following above for example: "# Note: query must return a vector/scalar single element response"

Reply:
Would be nice to have more explanation for the other fields as well